packaging/observability/flightctl-observability-reload.sh

```
@@ -0,0 +1,274 @@
+    
+    log "INFO" "All installed observability services are running correctly"
+}
+
+show_service_endpoints() {
+    log "INFO" "Observability service endpoints:"
+    
+    # Extract configuration values
+    local config_file="/etc/flightctl/service-config.yaml"
+    
+    if command -v yq >/dev/null 2>&1; then
+        local base_domain=$(yq e '.global.baseDomain // "localhost"' "$config_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)